Coverity does not use fallback version if it can't find a tag

**Describe the bug**
<!-- Please explain what happened and provide the context in which the bug occurred. -->
Caused by: java.lang.IllegalArgumentException: version can neither be null, empty nor blank

**Expected behavior**
<!-- Please tell us why you think the behavior is unexpected. If you can, please copy-paste logs or error messages you got while facing the bug. -->
Use some tag or don't submit, failing is not good

Should be fixable with `--always` flag
 
**How to reproduce**
<!-- Please provide us reproducible step-by-step guide on how to reproduce the bug. Please keep in mind that non-reproducible issues will be closed. -->
 
**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->
 
**Software version**
<!-- e.g. 3.1.4 -->
 
**Desktop (please complete the following information if relevant):**
 - OS: <!-- [e.g. iOS] -->
 - Browser: <!-- [e.g. chrome, safari] -->
 - Version: <!-- [e.g. 22] -->
 
**Additional context**
<!-- Add any other context about the problem here. -->

--- a/workflows/coverity.yaml
+++ b/workflows/coverity.yaml
@@ -20,7 +20,7 @@ jobs:
         run: sudo apt-get update && sudo apt-get install jq
 
       - name: Get version
-        run: printf "RELEASE_VERSION=%q\n" "$(git describe --tags)" >> $GITHUB_ENV
+        run: printf "RELEASE_VERSION=%q\n" "$(git describe --always --tags)" >> $GITHUB_ENV
 
       - name: Setup Maven Central
         uses: actions/setup-java@v4

